lib/membrane_mp4/muxer/isom.ex

```diff
@@ -96,25 +96,28 @@ defmodule Membrane.MP4.Muxer.ISOM do
   @impl true
   def handle_stream_format(
         Pad.ref(:input, pad_ref) = pad,
-        stream_format,
+        %type{} = stream_format,
         ctx,
         state
       ) do
-    cond do
+    case ctx.pads[pad].stream_format do
       # Handle receiving the first stream format on the given pad
-      is_nil(ctx.pads[pad].stream_format) ->
+      nil ->
         update_in(state, [:pad_to_track, pad_ref], fn track_id ->
           Track.new(track_id, stream_format, state.chunk_duration)
         end)
 
-      # Handle receiving all but the first stream format on the given pad,
-      # when stream format is duplicated - ignore
-      ctx.pads[pad].stream_format == stream_format ->
+      # Handle receiving a stream format of the same type
+      %^type{} ->
```

Comment:
Is it OK for us to accept change from `:avc3` to `:avc1`? Wouldn't it require appending something to the DCR?